microsoft-edge/webview2/get-started/winui.md

```diff
@@ -200,6 +200,28 @@ Edit the `MainWindow.xaml` and `MainWindow.xaml.cs` files to add a WebView2 cont
 1.  Close the app.
 
 
+<!-- ---------- -->
+#### WinAppSDK supports custom WebView2 environments
+
+Background information:
+
+WinAppSDK supports custom WebView2 environments.  See [WinUI3 WebView2 with a custom CoreWebView2Environment](https://github.com/microsoft/microsoft-ui-xaml/issues/6150).  WebView2 custom environment support has been added in WinAppSDK 1.5 (first appearing in [1.5.0-experimental2](/windows/apps/windows-app-sdk/experimental-channel#version-15-experimental-150-experimental2)).
+
+To implement a custom WebView2 environment, initialize WebView2 with one of the overrides of `WebView2.EnsureCoreWebView2Async` (listed below), and pass in your custom `CoreWebView2Environment` (and, optionally, custom `CoreWebView2ControllerOptions`):
```

Comment:
>implement [](http://example.com/codeflow?start=3&length=9)

Nit: activate (?)

Reply:
Is there a better word/wording than "implement" or "activate"?
Not sure this is a meaningful use of the word 'activate'.  PR has been already approved, so I hesitate to make this change that uses the word 'activate' in a way I haven't heard.
@david-risney @krschau

Reply:
instantiate